flybywiresim/a32nx
# fix(clock): make clock backlight subtle

Fixes #[issue_no]

## Summary of Changes


**Before:** The clock on the front instruments panel was initially connected to the default electric bus, which kept it turned off in cold and dark situations. 

**Now:** Since the electrical system implementation, it has been rightly connected to a DC Hotbus, and functions even when the rest of the aircraft is powered down. This gives an unrealistic look to the clock which appears to have a solid blue backlight when the aircraft is completely dark, as opposed to the references.

This PR fixes the strong blue backlight, and makes it more subtle, suitable with all lighting conditions.

## Screenshots (if necessary)



### BEFORE:
![image](https://user-images.githubusercontent.com/29005005/116477317-107d2900-a89a-11eb-959e-fe469cbc6096.png)
![image](https://user-images.githubusercontent.com/29005005/116477246-fa6f6880-a899-11eb-8226-e9512b037519.png)

### AFTER:
![image](https://user-images.githubusercontent.com/29005005/116477427-386c8c80-a89a-11eb-865f-43047ea9d3d5.png)
![image](https://user-images.githubusercontent.com/29005005/116477407-30ace800-a89a-11eb-838e-4a51b292b2c8.png)
![image](https://user-images.githubusercontent.com/29005005/116477505-5934e200-a89a-11eb-93e4-2c1a2d677a70.png)

## References


[IRL, the clocks on the 320neo apparently have similar LED panels instead of the LCD ones which commonly suffer from backlight bleeding. In addition to this, they also have an ambient light sensor which controls the brightness of the digits, and not the backlight itself. (Point 9: https://www.skalarki-electronics.com/en/a320-project/a320-clock-project.html)]

![image](https://user-images.githubusercontent.com/29005005/116477582-72d62980-a89a-11eb-875c-4b33b9db0f3e.png)
![image](https://user-images.githubusercontent.com/29005005/116477685-9305e880-a89a-11eb-8297-cc6c903cae9e.png)




## Additional context



Discord username (if different from GitHub): sidnov#8337

## Testing instructions


1. Spawn Cold and Dark in pitch dark environments. You can also adjust moonlight by skipping days in the weather panel.
2. Change different gates, lighting conditions, make sure the digits are visible enough.
3. Repeat similar steps for daylight conditions.
4. Clock functions should be unaffected, however, they can be tested.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
